widgetssdk/src/main/java/com/glia/widgets/view/VisitorCodeView.kt

```diff
@@ -145,23 +157,37 @@ class VisitorCodeView internal constructor(
     private fun applyRemoteConfigTheme() {
         Dependencies.getGliaThemeManager().theme?.callVisualizerTheme.apply {
             applyLayerTheme(this?.visitorCodeTheme?.background)
-            titleView.applyTextTheme(this?.visitorCodeTheme?.title)
+            successTitle.applyTextTheme(this?.visitorCodeTheme?.title)
+            // TODO apply correct theme when it becomes available
```

Comment:
I think it would be better to refer to the ticket for unified UI

Reply:
Done